plasmapy/formulary/tests/test_magnetostatics.py

```diff
@@ -55,6 +60,16 @@ def test_close_cw(self):
         assert np.all(np.isclose(B_cw.value, B_gw_cw.value))
         assert B_cw.unit == B_gw_cw.unit
 
+    def test_repr(self):
+        "Test __repr__ function"
+        gw_cw = self.cw.to_GeneralWire()
+        # round numbers to avoid calculation accuracy mismatch
+        gw_cw.t1 = -3.1516
+        gw_cw.t2 = +3.1516
+        assert repr(
+            gw_cw
+        ) == r"GeneralWire(parametric_eq=curve, t1=-3.1516, t2=3.1516, current=1.0A)"
```

Comment:
This could probably have been avoided by putting this as a doctest in the `__repr__` method; Pytest 5.1+ has capabilities to check for this sort of numerical discrepancy. It's okay to do it this way as well, though!